test/usage/bsb-native/package-lock.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "bsb-native",
+  "version": "0.1.0",
+  "lockfileVersion": 1,
+  "requires": true,
+  "dependencies": {
+    "bs-platform": {
+      "version": "github:bsansouci/bsb-native#d7e3f615218f0350a2755c4f2a20eb3047ae0dbd",
```

Comment:
I'll take a look at this tomorrow, and place a valid commit hash. I think I may have formatted it wrong.